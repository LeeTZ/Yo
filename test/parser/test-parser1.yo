a = 1-2;
if (1+2):{
    a=1;
}elif 2>2:{
	a = 2;
}
else:{
	a=3;
}
continue;
break;
return a;
return 3;