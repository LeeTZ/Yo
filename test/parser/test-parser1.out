a = 1 - 2
if 1 + 2:
a = 1
elif 2 > 2:
a = 2
elif 3 == 1:
a = 4
else:a = 3
continue
break
return 
return 3
while 1 > 0:
a = "5"
b = 7
for i = 1 to 3:
a = 1
b = 9
for i = 1 to 3:
a = 1
for i in a:
a[i] = 1
type a func a (int a
, int b
) a = 2, int b
